notebook/Dockerfile

```diff
@@ -1,34 +1,10 @@
-FROM alpine:3.8
-MAINTAINER Hail Team <hail@broadinstitute.org>
+FROM {{ service_base_image.image }}
+
```

Comment:
We should probably add the maintainer line back?

Reply:
Meh.  These are all internal.  Waste of time.